com.github.ovehbe.emailprinter/com.github.ovehbe.emailprinter.yml

```diff
@@ -0,0 +1,46 @@
+app-id: com.github.ovehbe.emailprinter
```

Comment:
as per the docmunentation https://docs.flathub.org/docs/for-app-authors/requirements#application-id  this application id is not acceptable.